vscode-dotnet-runtime-library/src/IDotnetListVersionsContext.ts

```diff
@@ -0,0 +1,54 @@
+/* --------------------------------------------------------------------------------------------
```

Comment:
Is this the right copyright header? Looks wrong to me.

Reply:
That's a very good point. There's an issue for it here: https://github.com/dotnet/vscode-dotnet-runtime/issues/905. every file has it, so Ill fix it in a different pr